db/data/queries/createTable.sql

```diff
@@ -1,16 +1,19 @@
+DROP DATABASE `quickOrder`;
+CREATE DATABASE `quickOrder`;
 ALTER DATABASE `quickOrder` COLLATE = `utf8_general_ci`;
 USE `quickOrder`;
 CREATE TABLE `Item` (
     `id` VARCHAR(20) NOT NULL PRIMARY KEY,
-    `img` VARCHAR(150),
+    `img` VARCHAR(100),
     `price` int
     
 );
 CREATE TABLE `Item_Trans`(
 	`id` VARCHAR(20) NOT NULL PRIMARY KEY, 
+    
     `lang`  VARCHAR(5) NOT NULL ,
-    `name` VARCHAR(40)NOT NULL ,
-    `description` VARCHAR(400),
+    `name` VARCHAR(20)NOT NULL ,
+    `description` VARCHAR(100),
```

Comment:
we discussed it's fine to increase the size of varchar to a higher number, did it get to 100 because you incorrectly resolved this conflict?

Because when writing description in English it might surpass 100 symbols

Reply:
I recover it to
    `name` VARCHAR(40)NOT NULL ,
    `description` VARCHAR(400),